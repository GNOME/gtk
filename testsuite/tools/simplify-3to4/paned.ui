<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk+" version="3.0"/>
  <object class="GtkPaned">
    <property name="visible">1</property>
    <child>
      <object class="GtkButton" id="button">
        <property name="visible">1</property>
      </object>
      <packing>
        <property name="resize">True</property>
        <property name="shrink">False</property>
      </packing>
    </child>
    <child>
      <object class="GtkLabel" id="label">
        <property name="visible">1</property>
      </object>
      <packing>
        <property name="resize">0</property>
        <property name="shrink">1</property>
      </packing>
    </child>
  </object>
</interface>
