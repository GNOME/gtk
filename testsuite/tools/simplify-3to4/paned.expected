<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk" version="4.0"/>
  <object class="GtkPaned">
    <property name="resize-end-child">0</property>
    <property name="shrink-start-child">0</property>
    <property name="start-child">
      <object class="GtkButton" id="button"/>
    </property>
    <property name="end-child">
      <object class="GtkLabel" id="label"/>
    </property>
  </object>
</interface>
